### PR TITLE
Update cmd and README

### PR DESCRIPTION
**Pull request by adrianchiris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+$ make build
+```
+Binary executable is located under `build` folder
+
+### Install
+
+Install MultiNetworkPolicy CRD into Kubernetes.
+
+```
+$ git clone https://gitlab-master.nvidia.com/adrianc/multi-networkpolicy-tc
```
Comment (rollandf) [inline README.md:48]: Please update repo to Mellanox

File: README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+$ cd multi-networkpolicy-tc
+$ kubectl create -f deploy/multi-net-crd.yaml
+customresourcedefinition.apiextensions.k8s.io/multi-networkpolicies.k8s.cni.cncf.io created
+```
+
+Deploy multi-networkpolicie-tc into Kubernetes.
+
+```
+$ git clone https://gitlab-master.nvidia.com/adrianc/multi-networkpolicy-tc
```
Comment (rollandf) [inline README.md:57]: Same here

File: README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Install MultiNetworkPolicy CRD into Kubernetes.
+
+```
+$ git clone https://gitlab-master.nvidia.com/adrianc/multi-networkpolicy-tc
+$ cd multi-networkpolicy-tc
+$ kubectl create -f deploy/multi-net-crd.yaml
+customresourcedefinition.apiextensions.k8s.io/multi-networkpolicies.k8s.cni.cncf.io created
+```
+
+Deploy multi-networkpolicie-tc into Kubernetes.
```
Comment (rollandf) [inline README.md:54]: Typo? `multi-networkpolicie-tc`

Comment (adrianchiris) [reply README.md:54]: yep, fixed.

File: README.md (hunk 2 above)
Comment (adrianchiris) [reply README.md:57]: fixed

File: README.md (hunk 1 above)
Comment (adrianchiris) [reply README.md:48]: fixed

File: pkg/tc/actuator_tc.go [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+	// list qdiscs
+	currentQDiscs, err := a.tcApi.QDiscList()
+	if err != nil {
+		return errors.Wrap(err, "failed to list qdiscs")
+	}
+
+	var ingressQDiscExist bool
+	for _, q := range currentQDiscs {
+		if q.Type() == types.QDiscIngressType {
+			ingressQDiscExist = true
```
Comment (ykulazhenkov) [inline pkg/tc/actuator_tc.go:33]: Do we need to add a break statement here?

File: pkg/tc/actuator_tc.go [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+	newFilterSet := NewFilterSetImpl()
+
+	for _, f := range existing {
+		existingFilterSet.Add(f)
+	}
+	for _, f := range objects.Filters {
+		newFilterSet.Add(f)
+	}
+
+	if existingFilterSet.In(newFilterSet) && existingFilterSet.Len() == newFilterSet.Len() {
```
Comment (ykulazhenkov) [inline pkg/tc/actuator_tc.go:84]: nit: add `Equal` method to FilterSet?

File: pkg/tc/actuator_tc.go (hunk 4 above)
Comment (adrianchiris) [reply pkg/tc/actuator_tc.go:33]: yes, will add.

File: pkg/tc/actuator_tc.go (hunk 5 above)
Comment (adrianchiris) [reply pkg/tc/actuator_tc.go:84]: ack, i also swapped the conditions to make returning false faster

Comment (adrianchiris) [reply pkg/tc/actuator_tc.go]: addressed in a separate PR #16

**Comment by adrianchiris** [comment]
@rollandf @ykulazhenkov PTAL. comments addressed

**Review by ykulazhenkov — APPROVED** [review]
LGTM
Need to rebase PR to main bracnh

**Review by rollandf — APPROVED** [review]

**Pull request merged**